pythran/pythonic/include/numpy/around.hpp

```diff
@@ -2,29 +2,48 @@
 #define PYTHONIC_INCLUDE_NUMPY_AROUND_HPP
 
 #include "pythonic/include/numpy/rint.hpp"
-#include <nt2/include/functions/pow.hpp>
+#include "pythonic/include/numpy/asarray.hpp"
+#include "pythonic/include/numpy/sum.hpp"
 
 namespace pythonic
 {
 
   namespace numpy
   {
+    namespace details
+    {
+      template <class T>
+      struct is_integral_valued {
+        // what a trick :-) use sum to get the dtype even when T is a scalar
+        // type
```

Comment:
Ok, I get it. It is funny, but not really easy to understand IMHO.
Something like : 

```
using type = typename std::conditional<is_numexp<T>, T::dtype, T>::type;
static constexpr bool value = std::is_integral<type>::value;
```

Make it easier to me to understand this part.